identity-service/src/audiusLibsInstance.js

```diff
@@ -35,4 +33,16 @@ async function initAudiusLibs () {
   return audiusInstance
 }
 
-module.exports = initAudiusLibs
+class AudiusLibsWrapper {
+  constructor () {
+    this.audiusLibsInstance = initAudiusLibs()
+  }
+
```

Comment:
you need an "async init" function here that's you move the contents of initAudiusLibs into. that should get awaited from configureAudiusInstance just like it does now. if you don't await it, there could be race conditions where libs isn't initialized

Reply:
Thanks, added an `async init()`